router/helpers_linux.go

```diff
@@ -0,0 +1,39 @@
+package router
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+)
+
+var kvpair = regexp.MustCompile(`(.*)=([^"].*|\"(.*)\")`)
```

Comment:
Try a regex like this, should simplify your code below a bit

https://play.golang.org/p/cUO1wi_BCj1

Reply:
Implemented and tested your recommended Regex.  Appears to work better, produces 3 results instead of 4 with an empty possibility which simplifies testing the response.